tools/BugReportTool/BugReportTool/Main.cpp

```diff
@@ -19,14 +19,15 @@ using namespace winrt::Windows::Data::Json;
 
 map<wstring, vector<wstring>> escapeInfo = {
     { L"FancyZones\\app-zone-history.json", { L"app-zone-history/app-path" } },
-    { L"FancyZones\\settings.json", { L"properties/fancyzones_excluded_apps" } }
+    { L"FancyZones\\settings.json", { L"properties/fancyzones_excluded_apps" } },
+    { L"PowerToys Run\\Settings\\QueryHistory.json", { L"Items/Query" }}
```

Comment:
Let's not include this file in the report.

Reply:
What are the reasons for that? We should include as much information as possible taking into account the privacy of users.

Reply:
The query history is private data, we should not include this type of data.
If we need that information for a very specific issue, we can always ask the user to upload the file.

Reply:
done